spritzle/resource/core.py

```diff
@@ -0,0 +1,33 @@
+#
+# spritzle/core.py
+#
+# Copyright (C) 2016 Andrew Resch <andrewresch@gmail.com>
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 3, or (at your option)
+# any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.    See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.    If not, write to:
+#   The Free Software Foundation, Inc.,
+#   51 Franklin Street, Fifth Floor
+#   Boston, MA    02110-1301, USA.
+#
+
+from aiohttp import web
+
+routes = web.RouteTableDef()
+
+
+@routes.get('/core/shutdown')
+async def get_shutdown(request):
+    def shutdown():
+        raise web.GracefulExit()
```

Comment:
I assume this runs the app on_shutdown handler?

Reply:
Yep.